### PR TITLE
PanelAR finalised version

### PR DESCRIPTION
**Pull request by czucca**
Hi Philip, 

I fixed everything you requested in #108 and tested it. This should be good to go. 

Claudia

### CODE REVIEW

File: R/extract.R [hunk 1]
```diff
@@ -5441,4 +5441,48 @@ extract.gnm <- function(model, include.aic = TRUE, include.bic = TRUE,
 }
 
 setMethod("extract", signature = className("gnm", "gnm"),
-          definition = extract.gnm)
\ No newline at end of file
+          definition = extract.gnm)
+
+# extention for panelAR objects (panelAR package)					 
+extract.panelAR <- function(model, include.rsquared = TRUE, include.nobs = TRUE, include.groups = TRUE,...){
```
Comment (leifeld) [inline R/extract.R]: Leave a space before `{`.

File: vignettes/texreg.Rnw [hunk 2]
```diff
@@ -439,7 +439,7 @@ Class			& Package			& Description \\
 \bottomrule
 \endfoot
 
-\caption[List of 100 supported model types (version 1.36.24).]{\label{tab:types} List of 100 supported model types (version 1.36.24).} \\ \\
+\caption[List of 103 supported model types (version 1.36.24).]{\label{tab:types} List of 100 supported model types (version 1.36.24).} \\ \\
```
Comment (leifeld) [inline vignettes/texreg.Rnw]: I think this needs to be changed twice in this line.

File: R/extract.R [hunk 3]
```diff
@@ -5441,4 +5441,48 @@ extract.gnm <- function(model, include.aic = TRUE, include.bic = TRUE,
 }
 
 setMethod("extract", signature = className("gnm", "gnm"),
-          definition = extract.gnm)
\ No newline at end of file
+          definition = extract.gnm)
+
+# extention for panelAR objects (panelAR package)
```
Comment (leifeld) [inline R/extract.R]: I noticed that you added all those recent extract functions to the end of the file. They all used to be in alphabetical order in the `extract.R` file. Can you rearrange that in alphabetical order when you get the chance?

**Comment by czucca** [comment]
All done!

File: R/extract.R [hunk 4]
```diff
@@ -2123,6 +2360,8 @@ setMethod("extract", signature = className("nlmerMod", "lme4"),
     definition = extract.nlmerMod)
 
 
+
+
```
Comment (leifeld) [inline R/extract.R]: Can you ensure there is equal spacing between functions everywhere?

File: R/extract.R [hunk 5]
```diff
@@ -2739,6 +3027,46 @@ setMethod("extract", signature = className("mnlogit", "mnlogit"),
     definition = extract.mnlogit)
 
 
+# extension for mhurdle objects (mhurdle package)
+extract.mhurdle <- function (model, include.nobs = TRUE, include.loglik = TRUE, ...)
+{
```
Comment (leifeld) [inline R/extract.R]: Put the `{` at the end of the previous line, with a space in between.

File: R/extract.R [hunk 6]
```diff
@@ -2739,6 +3027,46 @@ setMethod("extract", signature = className("mnlogit", "mnlogit"),
...
+    gof <- numeric()
+    gof.names <- character()
+    gof.decimal <- logical()
+    if (include.loglik == TRUE) {
+        gof <- c(gof, as.numeric(s$naive$logLik))
+        gof.names <- c(gof.names, "Log Likelihood")
+        gof.decimal <- c(gof.decimal, TRUE)
+    }
+    if (include.nobs == TRUE) {
+        gof <- c(gof, length(s$model[ ,1]))
```
Comment (leifeld) [inline R/extract.R]: Space before `1`. No space before the comma.

File: R/extract.R [hunk 7]
```diff
@@ -3197,6 +3589,100 @@ setMethod("extract", signature = className("ols", "rms"),
     definition = extract.ols)
 
 
+# extention for panelAR objects (panelAR package)					 
+extract.panelAR <- function(model, include.rsquared = TRUE, include.nobs = TRUE, include.groups = TRUE,...) {
+    s <- summary(model,...)
```
Comment (leifeld) [inline R/extract.R]: Add a space before `...`.

File: R/extract.R [hunk 8]
```diff
@@ -3197,6 +3589,100 @@ setMethod("extract", signature = className("ols", "rms"),
     definition = extract.ols)
 
 
+# extention for panelAR objects (panelAR package)					 
+extract.panelAR <- function(model, include.rsquared = TRUE, include.nobs = TRUE, include.groups = TRUE,...) {
```
Comment (leifeld) [inline R/extract.R]: Add a space before `...`.

File: R/extract.R [hunk 9]
```diff
@@ -4057,6 +4543,67 @@ setMethod("extract", signature = className("stergm", "tergm"),
     definition = extract.stergm)
 
 
+#extension for summary.lm objects (stats)
```
Comment (leifeld) [inline R/extract.R]: Space after `#`.

**Comment by czucca** [comment]
all done! very sorry for the formatting imprecisions!

**Pull request merged**